omod/src/main/java/org/openmrs/module/fhir/resources/FHIRConditionResource.java

```diff
@@ -24,17 +25,20 @@
 
 public class FHIRConditionResource extends Resource {
 
-	public Condition getByUniqueId(IdDt id) {
-		ConditionService conditionService = Context.getService(ConditionService.class);
-		Condition condition = conditionService.getCondition(id.getIdPart());
-		if (condition == null) {
-			throw new ResourceNotFoundException("Condition is not found for the given Id " + id.getIdPart());
-		}
-		return condition;
-	}
+    public Condition getByUniqueId(IdDt id) {
+        ConditionService conditionService = Context.getService(ConditionService.class);
```

Comment:
This is already existing service. Hence no need. I forgot to add it when doing appointment and added now